js/script.js

```diff
@@ -52,6 +87,12 @@ class UI {
     document.querySelector('#title').value = '';
     document.querySelector('#author').value = '';
     document.querySelector('#isbn').value = '';
+    document.querySelector('#authorEmail').value = '';
+    document.querySelector('#author-phone').value = '';
+  }
+
+  static show_error_text(fieldClass) {
+    document.querySelector(fieldClass).style.display = 'block';
```

Comment:
please use ids to identify elements. Use a class to update its style.
```suggestion
    document.getElementById(fieldId).classList.remove('hideError');
```